sdk/api/evaluation-task/create-single-turn.mdx

```diff
@@ -11,24 +11,42 @@ This method is versatile and can be used for two main scenarios:
 1.  **Test-Based Evaluation**: When you provide a `test_case_id`, Galtea evaluates your product's performance against a predefined challenge.
 2.  **Production Monitoring**: When you set `is_production=True` and provide an `input`, Galtea logs and evaluates real user interactions.
 
-### Example: Test-Based Evaluation
+
+### Example: Test-Based Evaluation with Standard and Custom Metrics
 ```python
+from galtea import Galtea, CustomScoreEvaluationMetric, EvaluationMetric
+
+# Define your custom metric
+class MyAccuracy(CustomScoreEvaluationMetric):
+    def __init__(self):
+        super().__init__(name="my-accuracy")
+    def measure(self, actual_output, **kwargs):
+        return 1.0 if "paris" in actual_output.lower() else 0.0
+
+custom_score_accuracy = MyAccuracy()
+
```

Comment:
I wrote it below also but here we need a metric.create() in case 'my-accuracy' does not exist